core/computed/entity-classification.js

```diff
@@ -20,11 +20,13 @@ class EntityClassification {
    */
   static makeUpAnEntity(entityCache, url) {
     if (!UrlUtils.isValid(url)) return;
-    // We can make up an entity only for those URLs with a valid domain attached.
-    // So we further restrict from allowed URLs to (http/https).
-    if (!Util.createOrReturnURL(url).protocol.startsWith('http')) return;
 
-    const rootDomain = Util.getRootDomain(url);
+    const parsedUrl = Util.createOrReturnURL(url);
+    const isChromeExtension = parsedUrl.protocol.startsWith('chrome-extension');
```

Comment:
nit: can this just be `=== 'chrome-extension:'` or am I missing something?

Reply:
It can be compared with `===`. I'll make the change.